deployment/postgres-init-db/sql/tables/merlin/external_events/external_event.sql

```diff
@@ -0,0 +1,81 @@
+create table merlin.external_event (
+    key text not null,
+    event_type_name text not null,
+    source_key text not null,
+    derivation_group_name text not null,
+    start_time timestamp with time zone not null,
+    duration interval not null,
+    properties jsonb,
+
+    constraint external_event_pkey
+      primary key (key, source_key, derivation_group_name, event_type_name),
+    constraint external_event_references_source_key_derivation_group
+      foreign key (source_key, derivation_group_name)
+      references merlin.external_source (key, derivation_group_name)
+      on delete cascade,
+    constraint external_event_references_event_type_name
+      foreign key (event_type_name)
+      references merlin.external_event_type(name)
+      on delete restrict
+);
+
+comment on table merlin.external_event is e''
+  'Externally imported events.';
+
+comment on column merlin.external_event.key is e''
+  'The key, or name, of the external_event.\n'
+  'Part of the primary key, along with the source_key, derivation_group_name, and event_type_name.';
+comment on column merlin.external_event.event_type_name is e''
+  'The type of this external_event.';
+comment on column merlin.external_event.source_key is e''
+  'The key of the external_source that this external_event is included in.\n'
+  'Used as a foreign key along with the derivation_group_name to directly identify said source.\n'
+  'Part of the primary key along with the key, derivation_group_name, and event_type_name.';
+comment on column merlin.external_event.derivation_group_name is e''
+  'The derivation_group that the external_source bearing this external_event is a part of.';
+comment on column merlin.external_event.start_time is e''
+  'The start time (in _plan_ time, NOT planner time), of the range that this source describes.';
+comment on column merlin.external_event.duration is e''
+  'The span of time of this external event.';
+comment on column merlin.external_event.properties is e''
+  'Any properties or additional data associated with this version that a data originator may have wanted included.\n'
+  'This column is used primarily for documentation purposes, and has no associated functionality.';
+
+create trigger check_external_event_duration_is_nonnegative_trigger
+before insert or update on merlin.external_event
+for each row
+when (new.duration < '0')
+execute function util_functions.raise_duration_is_negative();
+
+create function merlin.check_event_times()
```

Comment:
Minor: rename to `check_external_event_boundaries` -- `event`s are a separate concept in Aerie